test/yamls/v1/beta/pipelineruns/pipelinerun-with-final-tasks.yaml

```diff
@@ -0,0 +1,203 @@
+# Copied from https://github.com/tektoncd/catalog/blob/v1/git/git-clone.yaml :(
```

Comment:
I'm a bit confused-- it looks like this is the same pipeline as https://github.com/tektoncd/pipeline/blob/main/examples/v1/pipelineruns/pipelinerun-with-final-tasks.yaml. Why add it to yamls tests when it's already run in examples tests? Also, it doesn't look like either of these pipelines use any beta features.